.github/workflows/main.yaml

```diff
@@ -103,7 +103,7 @@ jobs:
         run: |
           npm install --global @cyclonedx/cyclonedx-npm
           cyclonedx-npm --omit dev --output-file node-sbom.json
-          wget https://github.com/CycloneDX/cyclonedx-cli/releases/download/v0.25.1/cyclonedx-linux-x64
+          wget https://github.com/CycloneDX/cyclonedx-cli/releases/download/v0.27.1/cyclonedx-linux-x64
```

Comment:
Is it really related to eslint upgrade?

Reply:
It's related to some of updated dependencies

Reply:
Is this problem?

Reply:
if it is not a mandatory upgrade to have eslint, it is better to have it in a separate Pull request to have a clearer separation. It can ease Pull request review and potential revert if we detect an issue later on.

Reply:
I can try to revert as much dependencies upgrades as possible, wdyt?

Reply:
it is fine for this time. The risk is relatively low. but a good habit when doing a new development (and not very complicated when thinking of doing it since beginning)